projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/boost-speed-score.tsx

```diff
@@ -110,7 +121,36 @@ const BoostSpeedScore = () => {
 					<>
 						<div className="mj-boost-speed-score__grade">
 							<span>{ __( 'Your website’s overall speed score:', 'jetpack-my-jetpack' ) }</span>
-							<span className="mj-boost-speed-score__grade--letter">{ speedLetterGrade }</span>
+							<span className="mj-boost-speed-score__grade--letter">
+								<button
+									onMouseEnter={ handleTooltipMouseEnter }
+									onFocus={ handleTooltipMouseEnter }
+									onMouseLeave={ handleTooltipMouseLeave }
+									onBlur={ handleTooltipMouseLeave }
+								>
+									{ speedLetterGrade }
+									{ isTooltipVisible && (
+										<Popover
+											placement={ isMobileViewport ? 'top-end' : 'right' }
+											noArrow={ false }
+											offset={ 10 }
+										>
+											<p className={ 'boost-score-tooltip__heading' }>
+												{ /* Add the `&nbsp;` at the end to prevent widows. */ }
+												{ __( 'Site speed performance:', 'jetpack-my-jetpack' ) }&nbsp;
+												{ speedLetterGrade }
+											</p>
+											<p className={ 'boost-score-tooltip__content' }>
+												{ __(
+													'You are one step away from making your site blazing fast. A one-second' +
```

Comment:
One more one-off comment, were we planning on having different copy depending on the users grade? When we did the score bars on the Jetpack dashboard we had several different copy options depending on what grade the site got

Reply:
Yes, in a follow-up PR that I'm working on now (will have it done soon), we are displaying various different copy, depending on the letter grade as well as varying states of the Boost card. See doc: https://docs.google.com/document/d/1o9jQiBBCrWDWeaICq7qhzFQ77zkH51zbyzat-IUOJEw/edit#heading=h.xz601n6x4pa7